torchvision/ops/misc.py

```diff
@@ -124,6 +116,105 @@ def __init__(
         self.out_channels = out_channels
 
```

Comment:
If I add a warning here. 
```
        warnings.warn(
            "Do not use ConvNormActivation class directly. "
            "Instead use torchvision.ops.Conv2dNormActivation instead.",
            FutureWarning,
        )
```


This will also get raised when user uses `Conv2dNormActivation` or `Conv3dNormActivation` as we are calling super() of this class.
I don't know about workaround to avoid raising such inherited warning.

Reply:
I was reviewing at the same time again... Github should add a feature to let you know someone is writing comments.

See my suggestion here: https://github.com/pytorch/vision/pull/5445#discussion_r814141497